tests/utils/streams/backends/test_kafka_config.py

```diff
@@ -0,0 +1,108 @@
+import importlib
+from snuba import settings
+from unittest import TestCase
+
+from snuba.datasets.storages import StorageKey
+from snuba.utils.streams.backends.kafka import get_default_kafka_configuration
+
+
+class TestKafkaConfig(TestCase):
```

Comment:
nit: up to you, I would not block for this. It seems you are not gaining a lot by defining the test as a class. You can just add the test functions in this module without declaring the class. 
(Then you can implement the teardown this way: https://docs.pytest.org/en/2.8.7/xunit_setup.html#method-and-function-level-setup-teardown)